### DISCUSSION TITLE
how to dynamically setting defaultLocale ,not in nuxtjs.config.js ?

### DISCUSSION THREAD

**Discussion by askie** [nuxt-modules/i18n#2623] [General]
### Describe the feature

How to dynamically specify the default language in the code of NuxtJS 2 instead of setting defaultLocale in nuxt.config.js. 
For instance, in js code dynamically setting it to fr so that after SSR rendering, the lang attribute in the HTML code is fr, and there is no /fr suffix in the URL.

**Comment by BobbieGoede** [comment]
Converted to a discussion as this is not an issue but a question.

I haven't worked on i18n v7 for Nuxt 2, but it likely also uses the `defaultLocale` to match and generate routes during build, that would mean it's not possible to dynamically change the `defaultLocale` at runtime.
